pkg/ytconfig/generator.go

```diff
@@ -690,3 +691,51 @@ func (g *Generator) GetDiscoveryConfig() ([]byte, error) {
 	}
 	return marshallYsonConfig(c)
 }
+
+func (g *Generator) getMasterCachesConfigImpl() (MasterCacheServer, error) {
+	spec := g.ytsaurus.Spec.MasterCaches
+	c, err := getMasterCachesCarcass(spec)
+	if err != nil {
+		return MasterCacheServer{}, err
+	}
+	g.fillCommonService(&c.CommonServer, &spec.InstanceSpec)
+	g.fillBusServer(&c.CommonServer, spec.NativeTransport)
+	c.Addresses = g.getMasterCachesAddresses()
+
+	return c, nil
+}
+
+func (g *Generator) GetMasterCachesConfig() ([]byte, error) {
```

Comment:
Please add a test for that method. There are a lot of tests of generators in generator_test.go file.

Reply:
Done